src/OfflineProducerHelper.cc

```diff
@@ -298,6 +298,10 @@ OfflineProducerHelper::eleBaseline (bigTree* tree, int iDau,
   bool isoS = (tree->combreliso->at(iDau) < relIso) || byp_isoS;
   bool idS = false;
   bool nISOidS = false;
+
+  // veto events with electrons in ECAL barrel-endcap transition region
+  //TODO: should be super cluster eta (not available in bigntuples at the moment)
+  bool etaGap = fabs(p4.Eta()) > 1.44 && fabs(p4.Eta()) < 1.57 && whatApply.Contains("etaGapVeto");
```

Comment:
Very minor: I would ask you to rename the variable to something like ```isInEtaGap```, so the huge condition below is more easily understandable.

Reply:
Please add a reference for this veto just above the bool. It might be useful as a future reference.

Reply:
Should we apply the gap on the veto?

Reply:
> Should we apply the gap on the lepton veto?

I thought about this as well, but apparently the electrons in the gap are just not used for analysis because they are not as well reconstructed, not necessarily fake. So I'd argue we also don't want additional leptons in the gap?

Reply:
Given what is written in the Twiki "SFs in the region 1.44 < |η| < 1.57, corresponding to the transition between the barrel and endcaps of ECAL, are not provided by the EGM POG since this region is excluded by most physics analyses." we should exclude this region also when we look for extra leptons

Reply:
Yup, after a chat with @bfonta I already changed that to also affect the third lepton veto